### PR TITLE
id:86060 simplify and move spike page from web app to brandgility embedded demo

### PR DESCRIPTION
**Pull request by wenzelua**

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: src/index.js [hunk 1]
```diff
@@ -1,7 +1,70 @@
...
+const app = {
+  init() {
+    this.clientPostMessageAPI = new ClientPostMessageAPI('.brandgility-iframe');
 
-render(null, root);
+    this.clientPostMessageAPI
+      .on('load', this.handleLoad)
+      .on('save', (id) => console.info('saved item id', id));
+
+    this.listenToClicks();
```
Comment (sunstorymvp) [inline src/index.js]: please rename to `attachListeners`

File: src/index.js [hunk 2]
```diff
@@ -1,7 +1,70 @@
...
+    document.querySelector('.openSavedItem').addEventListener('click', this.handleOpenSavedItem.bind(this));
+    document.querySelector('.openTemplate').addEventListener('click', this.handleOpenTemplate.bind(this));
+    document.querySelector('.saveButton').addEventListener('click', this.handleSave.bind(this));
+  },
+
+  handleLoad({ id, type }) {
+    document.querySelector('.loadedTemplateId').textContent = id;
+    document.querySelector('.loadedEntityType').textContent = type;
+
+    [...document.querySelectorAll('.control')].forEach((control) => {
```
Comment (sunstorymvp) [inline src/index.js]: please use `.forEach` on NodeList

File: src/index.js [hunk 3]
```diff
@@ -1,7 +1,70 @@
...
+    `;
+  },
+
+  clearInfo() {
+    document.querySelector('.loadedTemplateId').textContent = '';
+    document.querySelector('.loadedEntityType').textContent = '';
+    document.querySelector('.savedItemId').value = '';
+    document.querySelector('.templateId').value = '';
+
+    [...document.querySelectorAll('.control')].forEach((control) => {
```
Comment (sunstorymvp) [inline src/index.js]: same

File: src/index.html [hunk 4]
```diff
@@ -5,8 +5,54 @@
...
+    <div class="root">
+      <div class="controls">
+        <input type="text" class="control templateId" placeholder="template id">
+        <button type="button" class="control openTemplate">Open Template</button>
+        <input type="text" class="control savedItemId" placeholder="saved item id">
+        <button type="button" class="control openSavedItem">Open Saved Item</button>
+        <button type="button" class="control saveButton">Save</button>
+        <div>
+        <div>Current configuration id: <strong class="loadedTemplateId"></strong></div>
+          <div>Entity Type: <strong class="loadedEntityType"></strong></div>
```
Comment (sunstorymvp) [inline src/index.html]: see indendtation

File: src/index.html [hunk 5]
```diff
@@ -5,8 +5,54 @@
...
   </head>
   <body>
-    <div id="root"></div>
+    <div class="root">
+      <div class="controls">
+        <input type="text" class="control templateId" placeholder="template id">
+        <button type="button" class="control openTemplate">Open Template</button>
+        <input type="text" class="control savedItemId" placeholder="saved item id">
+        <button type="button" class="control openSavedItem">Open Saved Item</button>
+        <button type="button" class="control saveButton">Save</button>
```
Comment (sunstorymvp) [inline src/index.html]: please follow BEM class naming

**Comment by sunstorymvp** [comment]
👍

**Review by sunstorymvp — APPROVED** [review]

**Pull request merged**